examples/transform/src/main/java/org/apache/apex/examples/transform/POJOGenerator.java

```diff
@@ -37,24 +38,29 @@
  */
 public class POJOGenerator implements InputOperator
 {
+  public static final int MAX_YEAR_DIFF_RANGE = 30;
+
   @Min(1)
   private int maxCustomerId = 100000;
   @Min(1)
   private int maxNameLength = 10;
   @Min(1)
   private int maxAddressLength = 15;
-  private long tuplesCounter;
+  private long tuplesCounter = 0;
+
   // Limit number of emitted tuples per window
   @Min(1)
-  private long maxTuplesPerWindow = 100;
```

Comment:
maxTuplesPerWindow serves a different purpose, it controls a throughput rate without capping the total number of tuples. I think you need a separate setting.

Reply:
I initially went about using a separate variable but realised that the max tuples per window is essentially dead code.  It is not serving any purpose downstream and hence thought it can be removed. I can revert it back if you reckon there is some value add in this.

Reply:
To run the application standalone (not as test) and get a continuous stream of data, I would set the tuples per streaming window to establish a rate. That's different from controlling the total number of tuples emitted.